Documentation/go_client.md

```diff
@@ -0,0 +1,18 @@
+# etcd go client API
+
+This document describes tips for using the etcd go client (the package `github.com/coreos/etcd/client`).
+
+
+## Endpoint Selection Mode
+
+This section describes how to configure a policy of choosing a request destination node in etcd client programs. It will affects performance (latency and throughput) of request processing in the etcd cluster.
```

Comment:
The choice of destination node for client requests effects latency and throughput performance of etcd clusters.